test/controllers/eventResponse.test.js

```diff
@@ -0,0 +1,77 @@
+const Botmock = require('botkit-mock');
+const {expect} = require("chai");
+const commandController = require("../../skills/projectCommand.js");
+
+describe("Bot Response", () => {
+  beforeEach(()=>{
+    this.controller = Botmock({});
+
+    this.bot = this.controller.spawn({type: 'slack'});
+    commandController(this.controller);
+  });
+  afterEach(() => {
+    
+    this.controller.shutdown();
+  });
+  it('should test for response, Link added', () => {
+    return this.bot.usersInput([
+      {
+        type: 'slash_command',
+        user: 'someUserId',
+        channel: 'someChannel',
+        messages: [
+          {
+            text: 'add jira jira.com',
+            isAssertion: true,
+            command: '/project'
+          }
+        ]
+
+      }
+    ]).then((message) => {
+      console.log(message.text)
+        expect(message.text).to.equal("Link added")
+    })
+  })
+
+  it('should test for failure', () => {
+    return this.bot.usersInput([
+      {
+        type: 'slash_command',
+        user: 'someUserId',
+        channel: 'someChannel',
+        messages: [
+          {
+            text: 'add jira jira.com',
+            isAssertion: true,
+            command: '/project'
+          }
+        ]
+
+      }
+    ]).then((message) => {
+        expect(message.text).to.equal("Eggs")
```

Comment:
do you have it set up to spit out `eggs` on a failing input string?

Reply:
Nope that's why it fails. It's testing for failure.

Reply:
its saying a successful 'Link Added' should equal 'Eggs'

Reply:
Done, thanks.